### DISCUSSION TITLE
[WIKI] Missing https instead of http for proxy examples

### DISCUSSION THREAD

**Discussion by MalTeeez** [dani-garcia/vaultwarden#4123] [Q&A] [answered]
### Subject of the issue
The wiki proxy examples for nginx and nginx with path are missing an s, leading to 404s.
Seen at [https://github.com/dani-garcia/vaultwarden/wiki/Proxy-examples](https://github.com/dani-garcia/vaultwarden/wiki/Proxy-examples) at _proxy_pass http://vaultwarden-default;_
### Deployment environment
Nginx & Vaultwarden in Docker

* vaultwarden version: 1.30.1

* Reverse proxy and version: nginx-1.25.3

**Comment by MalTeeez** [comment]
To be noted: I am not 100% sure this is actually an issue, but it took me a while to find out that this was what was causing my 404, so maybe it is actually an issue. @BlackDex

**Comment by BlackDex** [comment]
Where is it missing an `s` according to you?
I use this exact same config for my self and my testing environment. No missing `s`.

If you mean https, then you have configured Vaultwarden to handle https by it self. While the config assumes Nginx will handle it.
Most of the time reverse proxies do the SSL offloading and forward the call unencrypted to the proxied application.

**Comment by MalTeeez** [comment]
Ah, realizing that now, that isn't needed with working proxy ssl. 
Would it be reasonable to note that in the examples since both have proxy ssl commented out by default.

The exact part I am talking about is this btw:
`      proxy_pass https://vaultwarden-default;`
instead of
`      proxy_pass http://vaultwarden-default;`

**Reply by BlackDex** [reply, marked as answer]
But proxy_pass shouldn't have https, since Vaultwarden isn't configured to do https by default.

If you configure Vaultwarden to do SSL by it self to, then yes you have to do that. But none of the examples do this.

So, i don't really see the need into adding this. It would only make it more complex.
